vendor/github.com/goji/httpauth/.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+sudo: false
+
+matrix:
+  include:
+    - go: 1.2
+    - go: 1.3
+    - go: 1.4
+    - go: 1.5
```

Comment:
Oh, I didn't notice that it is for goji... So next time please separate adding dependencies and your own codes into two PR...